src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/ReleaseDateTypes.kt

```diff
@@ -13,7 +13,7 @@ class ReleaseDateTypes(
     if (sentence.isCalculationInitialised()) {
       val underEighteenAtTimeOfRelease = offender.getAgeOnDate(sentence.sentenceCalculation.releaseDate) < INT_EIGHTEEN
       val lessThanTwelveMonths = sentence.durationIsLessThan(12, MONTHS)
-      if (underEighteenAtTimeOfRelease && lessThanTwelveMonths) {
+      if (underEighteenAtTimeOfRelease && lessThanTwelveMonths && sentence !is DetentionAndTrainingOrderSentence) {
```

Comment:
Could this condition be moved higher up. Alongside isCalculationInitialised(). Also should the check be sentence is StandardSentence. (I think this only applies to sds sentences, rather than anything thats not DTO).